docs/terminal/getting-started.md

```diff
@@ -0,0 +1,156 @@
+
+## Run your first command in the terminal
+
+While you're writing code, you might have to run shell commands to build, test, or deploy your application. The integrated terminal in VS Code enables you to run these commands without leaving the editor.
+
+To get started with the terminal:
+
+1. Start VS Code and open a folder or workspace.
+
+1. Open the terminal by selecting **View** > **Terminal** from the menu bar, or by pressing the `kb(workbench.action.terminal.toggleTerminal)` keyboard shortcut.
+
+    Based on your operating system configuration, the terminal opens with a default shell like Bash, PowerShell, or Zsh. The shell's working directory starts at the root of the workspace folder.
+
+    ![Open the terminal](./images/getting-started/open-terminal.png)
+
+1. Enter a basic command like `ls` to list the files in the current directory.
+
+    The terminal displays the output of the command, similar to a standalone terminal, except that you stay within the editor.
+
+    ![Run a command in the terminal](./images/getting-started/terminal-output.png)
+
+    > **Tip**: You can enlarge the terminal by dragging the terminal panel's border or by selecting the `^` icon to maximize the panel.
+
+## Interact with command output
+
+The terminal in VS Code also provides features to interact with command output. Commands often output file paths or URLs that you might want to open or navigate to. For example, a compiler or linter might return an error message with a file path and line number. Instead of searching for that file, you can select the link in the terminal output to open the file directly in the editor.
+
+Let's see how you can interact with the command outputs in the terminal:
+
+1. Open the terminal where you previously ran the `ls` command.
+
+1. In the terminal, hold the `kbstyle(Ctrl)`/`kbstyle(Cmd)` key, hover over a file name, and then select the link.
+
+    Notice that when you hover over text in the output, it changes into a link. When you select a file name, VS Code opens the selected file in the editor.
+
+    ![Navigate to files/URLs using links in terminal output](./images/getting-started/terminal-links.png)
+
+    All text in the terminal output is clickable. If you select a hyperlink in the terminal, it opens the link in the default browser. For other text, VS Code tries to search the workspace for files that contain the text.
+
+1. Run the following command to create a `Command.txt` file that contains a list of available shell commands.
+
+    * PowerShell
+
+        ```powershell
+        Get-Command | Out-File -FilePath .\Command.txt
+        ```
+
+    * Bash
+
+        ```bash
+        ls -l /usr/bin > Command.txt
+        ```
+
+1. Enter the following command to search for a command in the `Command.txt` file.
+
+    * PowerShell
+
+        ```powershell
+        Get-ChildItem *.txt | Select-String "dir"
+        ```
+
+    * Bash
+
+        ```bash
+        grep -n "dir" *.txt
```

Comment:
Should these use Command.txt explicitly? There may be other .txt files

Reply:
It's not needed but its a quick way to set the user up for success in this tutorial. Unless we get people to download some predefined repo, we don't know what files are available. And then we'd need to provide a more vague search `grep -n <some text> <your files>`.